tests/validation/tests/v3_api/common.py

```diff
@@ -2962,3 +2917,61 @@ def update_and_validate_kdm(kdm_url, admin_token=ADMIN_TOKEN,
     kdm_refresh_url = rancher_api_url + "/kontainerdrivers?action=refresh"
     response = requests.post(kdm_refresh_url, verify=False, headers=header)
     assert response.ok
+
+
+def prepare_hardened_nodes(aws_nodes, profile, node_roles, 
+client=None, cluster=None, custom_cluster=False):
+    i = 0
+    if profile == 'rke-cis-1.4':
+        for aws_node in aws_nodes:
+            aws_node.execute_command("sudo sysctl -w vm.overcommit_memory=1")
+            aws_node.execute_command("sudo sysctl -w kernel.panic=10")
+            aws_node.execute_command("sudo sysctl -w kernel.panic_on_oops=1")
+            if "etcd" in node_roles[i]:
+                aws_node.execute_command("sudo useradd etcd")
+            if custom_cluster:
+                docker_run_cmd = \
+                get_custom_host_registration_cmd(client,
+                                                 cluster,
+                                                 node_roles[i],
+                                                 aws_node)
+                aws_node.execute_command(docker_run_cmd)
+            i += 1
+    elif profile == 'rke-cis-1.5':
+        for aws_node in aws_nodes:
+            aws_node.execute_command("sudo sysctl -w vm.overcommit_memory=1")
```

Comment:
Is it possible to move these hardening commands to a script and either run the script as a whole or line by line?